python/hail/tests/tests_v0_1_cf0e0137a.py

```diff
@@ -860,11 +863,12 @@ def test_concordance(self):
     def test_hadoop_methods(self):
         data = ['foo', 'bar', 'baz']
         data.extend(map(str, range(100)))
+        data = [bytearray(d, encoding="utf8") for d in data]
```

Comment:
This seems potentially bad. What went wrong with the original code here? If people are going to be expected to manually construct byte arrays every time they use these functions, we need to modify the functions.